app/views/articles/_user_metadata.html.erb

```diff
@@ -43,43 +43,47 @@
           </div>
         </li>
       <% end %>
-      <% if @user.employment_title.present? %>
-        <li>
-          <div class="key">
-            Work
-          </div>
-          <div class="value">
-            <%= @user.employment_title %>
-            <% if @user.employer_name.present? %>
-              <span class="opacity-50"> at </span>
-              <% if @user.employer_url.present? %>
-                <a href="<%= @user.employer_url %>" target="_blank" rel="noopener"><%= @user.employer_name %></a>
-              <% else %>
-                <%= @user.employer_name %>
+      <% if ForemInstance.dev_to? %>
```

Comment:
@benhalpern Should we slowly phase out these special cases?

We have the profile fields and their information, we can still render them just like we render any locally defined profile fields (i.e. on the profile page) but maybe we get rid of special treatment for DEV for the right sidebar?

Reply:
I much prefer `employment_title` being special vs DEV being special. So definitely in favor of removing `ForemInstance.dev_to?` as a solution of any kind.

Employment title is somewhat universal and may be present in any professional network.

_However_, if that leaves a bad taste in your mouth from a complexity perspective..... We could ditch all these fields and run a migration where they logically fall into one single "Work" field for display.

So this:

<img width="262" alt="Screen Shot 2021-06-17 at 6 35 11 PM" src="https://user-images.githubusercontent.com/3102842/122480384-c3186080-cf9a-11eb-87a3-c40a517378f2.png">

Becomes this:

<img width="283" alt="Screen Shot 2021-06-17 at 6 36 16 PM" src="https://user-images.githubusercontent.com/3102842/122480487-ed6a1e00-cf9a-11eb-8fd1-512ea53efc34.png">

Basically we migrate to a single work field which takes an arbitrary string. That's pretty okay to me. We don't lose a lot if we do that right.

Reply:
@benhalpern Thanks for the feedback so far! Just to clarify: would the new `work` field be 

1. A `ProfileField` on DEV only (which means we have to create it and it gets stores as JSONB in the `data` column of the profiles) or
2. available on all Forems in which case we'd add it as a column to the `profiles` table.

Reply:
@citizen428 This can be a custom profile field, so for compatibility purposes it should be migrated to for _existing_ Forems (not just DEV), as a matter of maintaining backwards compatibility for those who have been using `employer_title,name,url`... So we should migrate data to this column not only on DEV, but we don't need to make "work" a base profile field like `location`. 

Here is a screenshot from the.community.club where folks use the employment trifecta at the moment)...

<img width="1149" alt="Screen Shot 2021-06-19 at 5 36 12 PM" src="https://user-images.githubusercontent.com/3102842/122655986-d8b99180-d124-11eb-9060-1c55434e8d32.png">

Reply:
@benhalpern Agreed, a single "Work" field with migrated data makes sense. Just so we are 100% on the same page, we essentially have two options here:

1. We add this as a column to `profiles`, same as `location`, `summary` and `website_url`. This would guarantee that the field always exists so we can use it in more contexts but admins can't disable it.
2. We add it as a `ProfileField`. This would give admins the option to remove it which means we could only use it in very limited contexts, like the profile page itself.

Please let me know what your preference is here.

Reply:
@citizen428 Let's go with **Option 2**. We'll migrate to a `ProfileField` which is for display on the profile, and no guarantee that it is present on any Forem.

It is for profile display-purposes only and/or any additional functionality that is generally applied to `ProfileField` in the future.